src/config.development.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   RELEASE_ID: 'development',
   CODE_VERSION: 'development',
+  DEPLOYED_ENV: 'development',
```

Comment:
the heroku environments run in development but it'd be nice to be able to differentiate them in the logs. will we be supplying `hostname` in any other metadata? can we supply `APP_ENV` as metadata separately and not override `DEPLOYED_ENV` here?